Dockerfile

```diff
@@ -1,35 +1,28 @@
-FROM debian
-MAINTAINER Andreas Stenius git@astekk.se
+FROM alpine:3.4
+MAINTAINER Marco Wessel <marco@driebit.nl>
 
-ENV DEBIAN_FRONTEND noninteractive
-
-# httpredir.debian.org/debian fails too much, so replace it with a fixed mirror.
-RUN echo \
-   'deb ftp://ftp.nl.debian.org/debian/ jessie main\n \
-    deb ftp://ftp.nl.debian.org/debian/ jessie-updates main\n \
-    deb http://security.debian.org jessie/updates main\n' \
-    > /etc/apt/sources.list
+ADD . /opt/zotonic
+WORKDIR /opt/zotonic
```

Comment:
I don't care if the build is faster, I care about the resulting image size which would balloon if I did that.

I will leave this as it is.